### ISSUE TITLE
Option to re-download blobs of my videos?

### ISSUE THREAD

**Issue by m93a** [lbryio/lbry-desktop#5493]
I uploaded a few videos to LBRY and after that I had to delete my blobfiles because I didn't have enough space on my disk. Now I bought a larger disk and I'd like to redownload the blobs, so that I can help with the distribution of my content. However, it seems that the only way to download the blobfiles again is to watch each one of my videos start to end. An option to bulk-download them would be neat.

**Comment by tzarebczan** [comment]
Thanks for the issue. There's a separate ticket filed for bulk downloading from a channel. 

If you go to your channel page and hover over the thumbnail, you should see a download button in the top right. Or you can also go into the page and click the download button. 

Make sure your download > advanced settings include hosted data.

**Comment by m93a** [comment]
Do you think you could eli5 how does file hosting work then? I thought I can seed only the blobs, not the downloaded .mp4 videos. Or does clicking on Download get me both a playable file and the blobfiles? Thanks!

**Comment by tzarebczan** [comment]
It depends on what your settings are in the app, see https://lbry.com/faq/host-content

But i checked and looks like the download option forces both mp4s and blobs at this time.
